docs/content-ai/index.html

```
@@ -0,0 +1,266 @@
+  subgraph you["YOUR APP — owns I/O & review"]
+    direction TB
+    C["AiContract / IngestContract<br/><span style='font-size:11px'>schema · fields · presets</span>"]
+    CFG["AiConfig<br/><span style='font-size:11px'>provider · model</span>"]
+    AS["assemble()<br/><span style='font-size:11px'>store + event-log writes</span>"]
+    EV["events + sinks<br/><span style='font-size:11px'>suppression · tracing</span>"]
+  end
+  subgraph sub["content-ai — owns orchestration, NO I/O"]
+    direction TB
+    R["runFill · runRefine · runRefresh"]
+  end
+  C --> R
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)